### ISSUE TITLE
Application can't be found anymore in PlayStore

### ISSUE THREAD

**Issue by KB4284848** [mashlol/notify#53]
Hi, it looks like it has been sadly removed from PlayStore as link in Readme isn't working anymore.
Was it by Google? I can still receive notifications from app however.

**Comment by mashlol** [comment]
Yes Google removed it due to "collecting advertising ID" although I have no idea what is collecting that - I already turned off the firebase automatic collection of advertising ID. Some other google project is probably collecting it automatically...I'll try to get it back up when I can

**Comment by RealDense** [comment]
Is there another way to get it since it is no longer on Google Play? I just heard about this app and I would love to have it on my phone.

**Comment by jmarin217** [comment]
You could build it yourself if there aren't any mirrors anywhere.

**Comment by Darkyere** [comment]
Yeah i wold like to see it back aswell.
Wonderfull little app that tells me my pc is still up and running.
Great work btw.

**Comment by jikamens** [comment]
FWIW, given that the app has disappeared from the app store and notifications from the CLI are no longer working, I decided to see if I could switch to using IFTTT to send messages from Linux to my Android phone. Indeed, it appears that you can. You can set up an IFTTT applet which accepts HTTPS requests using the Webhooks service and transmits notifications to the IFTTT app on your phone using the Notifications service. The incoming Webhooks requests can set variables for, e.g., the notification title and content, and the Notifications service can then access those variables to transmit the content to your phone. Just mentioning this here in case it proves useful to someone else.

**Comment by morpheusthewhite** [comment]
Thia may help too 

https://blog.usejournal.com/how-to-fix-advertising-id-policy-violation-in-google-play-store-6d9cf92d335d

**Comment by stellarpower** [comment]
W.R.T. the play store, would F-Droid be a viable alternative in the interim?

**Comment by jmarin217** [comment]
Maybe on a separate repository, the part where it relies on a central
server is an anti-feature. Unless I completely misunderstood how this
project works.

**Comment by jmarin217** [comment]
In the meantime I've found wirepusher to be extremely helpful (on the play
store)

**Comment by ctankutay** [comment]
you can download  notify-android-app.apk  for android phones via:

[https://drive.google.com/drive/folders/16WFx1qz6uMu1QWyL4pEVap6Ov8-5skn0?usp=sharing](url)

![notify android app token screen](https://user-images.githubusercontent.com/55889263/65785166-70f73200-e15c-11e9-8c7b-f72daf0a2b6a.jpg)

**Comment by mukheshpugal** [comment]
Thanks @ctankutay . This works
